src/pages/[platform]/build-a-backend/data/custom-business-logic/index.mdx

```diff
@@ -510,6 +510,40 @@ safePrint(echoResponse.echo.content);
 ```
 
 </InlineFilter>
+## Supported Argument Types in Custom Operations
+
+Custom operations can accept different types of arguments. Understanding these options helps define flexible and well-structured APIs.
+
+### Defining Arguments in Custom Operations
+
+When defining a custom operation, you can specify arguments using different types:
+
+- **Scalar Fields**: Basic types such as `string`, `integer`, `float`, etc.
+- **Custom Types**: Define inline `customType`.
+- **Reference Types**: Use `a.ref()` to reference enums and custom types.
```

Comment:
```suggestion
- **Custom Types**: Define inline `customType`
- **Reference Types**: Use `a.ref()` to reference enums and custom types
```

nit, we can omit the periods in unordered lists when we are not writing complete sentences or paragraphs